core/no_check_certificate_deserializer.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+use std::fmt::{self, Formatter};
+use std::marker::PhantomData;
+
+use serde::{de, Deserialize, Deserializer};
+
+pub fn deserialize_no_check_certificate<'de, D>(
```

Comment:
This deserializer is used at 2 places: extensions/net and extensions/fetch. Similar to the NoCertificateVerification might be just moved somewhere else.